lib/osf-components/addon/components/registries/my-registrations-list/registrations/template.hbs

```diff
@@ -0,0 +1,26 @@
+<ContentPlaceholders as |placeholder|>
+    <PaginatedList::HasMany
+        @model={{@user}}
+        @relationshipName='registrations'
+        @usePlaceholders={{false}}
+        as |list|
+    >
+        <list.item as |registration|>
+            {{#if registration}}
+                <NodeCard
+                    @node={{registration}}
+                    @showTags={{true}}
+                />
+            {{else}}
+                {{placeholder.text lines=1}}
+            {{/if}}
+        </list.item>
+        <list.empty>
+            <p
+                data-test-registration-list-none
+            >
+                {{t 'registries.myRegistrationsList.noRegistrations'}}
+            </p>
```

Comment:
I think some UX Consult feedback would go a long way for these "No Drafts/Registrations" messages.

Reply:
We might could get it done during demo time today.